src/index.js

```diff
@@ -1,35 +1,50 @@
 // import _ from 'lodash';
+/* eslint-disable prefer-destructuring */
 import './style.css';
+import Status from './task-status';
 
-const task1 = {
+const dataStorage = JSON.parse(localStorage.getItem('todoTasks'));
+
+let task1 = {
   description: 'Go to the supermarket',
   completed: false,
   index: 0,
 };
 
-const task2 = {
+task1 = dataStorage[0];
```

Comment:
- [ ] <b>REQUIRED:</b> I tried to run the app, but it render an empty page with the following error in the console. kindly fix it.
![Screen Shot 2021-09-22 at 21 12 00](https://user-images.githubusercontent.com/45217364/134407236-f5982624-9531-4f63-94cd-057a60a53b6f.png)

Reply:
Hi @rukundoeric , thanks! I didn't notice that before, I have solved it. Could you help me and tell me if know it's running correctly and properly before submitting another pull request review! Only if you have time, don't worry!

Thank's for your time! 😄

Reply:
Sorry, I didn't get the message on time. but no worries I'm reviewing it.